core/language/en-GB/Snippets/ExtJSReadme.tid

```diff
@@ -0,0 +1,15 @@
+title: $:/language/Snippets/ExtJSReadme
```

Comment:
I don't understand, why the additional docs is part of `$:/language/` without language code eg: en-GB here

Reply:
The ExtJSReadme should eventually go to TW5.com edition. Since this is a draft PR, I feel it is easier to view the doc if everything is contained in this PR generated wikis.

GetTiddlyWikiJS is meant to be contained in every wiki to facilitate users in acquiring core JS depending on their particular setups. What would be a better title path?

Reply:
When docs updates are required to accompany a functional change, then we push the docs change to `master` along with the code changes. So, please refactor the docs to go into the `tw5.com` edition